### ISSUE TITLE
Server downloads inactive kontainer-engine cluster drivers

### ISSUE THREAD

**Issue by erikwilson** [rancher/rancher#17595]
**What kind of request is this (question/bug/enhancement/feature request):**
Bug

**Steps to reproduce (least amount of steps as possible):**
Perform fresh install of air-gapped server (no management-state)

**Result:**
Errors like:
```
2019/01/24 13:40:18 [INFO] Download https://github.com/rancher/kontainer-engine-driver-aliyun/releases/download/v0.2.1/kontainer-engine-driver-aliyun-linux
2019/01/24 13:40:18 [INFO] Download https://github.com/rancher/kontainer-engine-driver-huawei/releases/download/v0.1.0/kontainer-engine-driver-huawei-linux
2019/01/24 13:40:18 [INFO] Download https://github.com/rancher/kontainer-engine-driver-tencent/releases/download/v0.2.0/kontainer-engine-driver-tencent-linux
2019/01/24 13:40:19 [ERROR] KontainerDriverController aliyunkubernetescontainerservice [mgmt-kontainer-driver-lifecycle] failed with : Get https://github.com/rancher/kontainer-engine-driver-aliyun/releases/download/v0.2.1/kontainer-engine-driver-aliyun-linux: dial tcp: lookup github.com: no such host
2019/01/24 13:40:19 [ERROR] KontainerDriverController tencentkubernetesengine [mgmt-kontainer-driver-lifecycle] failed with : Get https://github.com/rancher/kontainer-engine-driver-tencent/releases/download/v0.2.0/kontainer-engine-driver-tencent-linux: dial tcp: lookup github.com: no such host
2019/01/24 13:40:19 [ERROR] KontainerDriverController huaweicontainercloudengine [mgmt-kontainer-driver-lifecycle] failed with : Get https://github.com/rancher/kontainer-engine-driver-huawei/releases/download/v0.1.0/kontainer-engine-driver-huawei-linux: dial tcp: lookup github.com: no such host
```

**Other details that may be helpful:**
Cluster drivers are marked as `Inactive`

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI):
master (2decf9df)
- Installation option (single install/HA):
air-gap

**Comment by daxmc99** [comment]
@erikwilson  Just to make sure, the preferred behavior would be to *not* download inactive cluster drivers?

**Comment by jiaqiluo** [comment]
The big fix is validated on Rancher:v2.2.0-rc1 on HA airgap installation mode.

Checked Rancher's log and did not see the KontainerDriverController related error messages about inactive node drivers.
